estate/models/estate_property.py

```diff
@@ -0,0 +1,32 @@
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(required=True, string="Title")
+    description = fields.Text(string="Description")
+    postcode = fields.Char(string="Postcode")
+    date_availability = fields.Date(default=fields.Date.add(fields.Date.today(), days=90), copy=False, string="Available From")
+    expected_price = fields.Float(required=True, string="Expected Price")
+    selling_price = fields.Float(readonly=True, copy=False, string="Selling Price")
+    bedrooms = fields.Integer(default=2, string="Bedrooms")
+    living_area = fields.Integer(string="Living Area (sqm)")
+    facades = fields.Integer(string="Facades")
+    garage = fields.Boolean(string="Garage")
+    garden = fields.Boolean(string="Garden")
+    garden_area = fields.Integer(string="Garden Area (sqm)")
+    garden_orientation = fields.Selection(
+        string="Garden Orientation",
+        selection=[("north", "North"), ("south", "South"), ("east", "East"), ("west", "West")],
+        help="Type to detect orientation of the garden")
+    active = fields.Boolean(default=True)
+    state = fields.Selection(
+        default='new',
+        selection=[
+            ('new', 'New'),
+            ('offer received', 'Offer Received'),
+            ('offer accepted', 'Offer Accepted'),
```

Comment:
By convention we use pyhon-like style.
```suggestion
            ('offer_received', 'Offer Received'),
            ('offer_accepted', 'Offer Accepted'),
```